### ISSUE TITLE
Hypertension resistant to conventional therapy

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10217]
**Preferred term label:**
Hypertension resistant to conventional therapy

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C1834155

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Various aldosterone-producing disorders
Fibromuscular dysplasia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000001
- label: Hypertension resistant to conventional therapy
- parents: HP:0000822
- pmid: PMID:30354828
- def: Resistant hypertension is defined as above-goal elevated blood pressure (BP) in a patient despite the concurrent use of 3 antihypertensive drug classes, commonly including a long-acting calcium channel blocker, a blocker of the renin-angiotensin system (angiotensin-converting enzyme inhibitor or angiotensin receptor blocker), and a diuretic.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10217
- synonyms: Resistant hypertension(NONE)
